project-koku/koku
# [COST-5133 + COST-5134] - Add support for OCP/(GCP/Azure) trino updates

## Jira Ticket

[COST-5133](https://issues.redhat.com/browse/COST-5133)
[COST-5134](https://issues.redhat.com/browse/COST-5134)

## Description
This change will enable processing of OCP on Azure and GCP data via trino instead of data frame wrangling. This is gated behind unleash.

## Testing

1. Bring up koku
2. Go to unleash and clone the schema flag template to a new flag called `cost-management.backend.feature-cost-5129-ocp-cloud-processing` and enable it.
3. Load your openshift source data
4. Load your {Azure/GCP} source data that goes with the OCP source from above
5. Verify in trino that the existing `aws_openshift_daily` table and the `managed_aws_openshift_daily` table have the same data. Some example queries below:
Column Counts:
```
trino:org1234567> SELECT count(*) FROM azure_openshift_daily;
 _col0 
-------
 15431 
(1 row)

trino:org1234567> SELECT count(*) FROM managed_azure_openshift_daily ;
 _col0 
-------
 15431 
(1 row)
```


## Release Notes

```markdown
* [COST-####](https://issues.redhat.com/browse/COST-####) Fix some things
```
